pandas/tests/series/indexing/test_delitem.py

```diff
@@ -50,3 +51,23 @@ def test_delitem_missing_key(self):
 
         with pytest.raises(KeyError, match=r"^0$"):
             del s[0]
+
+    def test_delitem_extension_dtype(self):
```

Comment:
maybe worthile adding all EA types here?

Reply:
ill add a test in the extension tests next pass